rockcraft/extensions/expressjs.py

```diff
@@ -114,15 +114,15 @@ def _check_project(self) -> None:
             or "start" not in self._app_package_json["scripts"]
         ):
             raise ExtensionError(
-                "missing start script",
+                "missing start script in package.json",
```

Comment:
What do you think about:


```suggestion
                "missing 'scripts.start' field in package.json",
```